### PR TITLE
Add company and address fields to attendees

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

**Comment by raphaelm** [comment]
Hmm, Travis fails for obscure reasons. Test pass locally, so ready for review

File: src/pretix/control/templates/pretixcontrol/event/settings.html [hunk 1]
```diff
@@ -83,12 +83,16 @@ <h1>{% trans "General settings" %}</h1>
             <fieldset>
                 <legend>{% trans "Attendee data" %}</legend>
                 {% bootstrap_field sform.attendee_names_asked layout="control" %}
-                {% bootstrap_field sform.attendee_names_required layout="control" %}
+                {% bootstrap_field sform.attendee_names_required lddressformayout="control" %}
```
Comment (pc-coholic) [inline src/pretix/control/templates/pretixcontrol/event/settings.html:86]:
```suggestion
                {% bootstrap_field sform.attendee_names_required layout="control" %}
```

I guess, the `ddressform` (`addressform`?)-leftover is not needed anymore at all...

File: src/pretix/base/forms/questions.py [hunk 2]
```diff
@@ -262,6 +262,73 @@ def __init__(self, *args, **kwargs):
                     }
                 )
             )
+        if item.admission and event.settings.attendee_company_asked:
+            self.fields['company'] = forms.CharField(
+                required=event.settings.attendee_company_required and not self.all_optional,
+                label=_('Company'),
+                initial=(cartpos.company if cartpos else orderpos.company),
+            )
```
Comment (pc-coholic) [inline src/pretix/base/forms/questions.py:270]: Could we move this up to lines 245 to have the same layout as for our other address-forms?

Company
Name
Address

File: src/pretix/presale/templates/pretixpresale/event/fragment_cart.html [hunk 3]
```diff
@@ -44,12 +44,51 @@
...
+                    {% if line.item.admission and event.settings.attendee_company_asked %}
+                        <dt class="sr-only">
+                            {% trans "Attendee company" %}
+                        </dt>
+                        <dd class="toplevel">
+                            <span data-toggle="tooltip" title="{% trans "Attendee company" %}">
+                                {% if line.company %}{{ line.company }}{% else %}<em>{% trans "not answered" %}</em>{% endif %}
+                            </span>
+                        </dd>
+                    {% endif %}
```
Comment (pc-coholic) [inline src/pretix/presale/templates/pretixpresale/event/fragment_cart.html:75]: Same question here with regards to the order of the "address-block" - although imho not that necessary as in the address-collection form

**Pull request merged**